### PR TITLE
added styles to sign-in, sign-up

### PR DESCRIPTION
**Pull request by AntonBurduzha**
#2

### CODE REVIEW

**Comment by AntonBurduzha** [comment]
Angular material will be changed to UIKit v.2

File: app/app.module.js [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+import 'uikit/dist/fonts/fontawesome-webfont.ttf';
+import 'uikit/dist/fonts/fontawesome-webfont.woff2';
+import 'uikit/dist/fonts/fontawesome-webfont.woff';
+
+import 'uikit/dist/css/components/datepicker.min.css';
+import 'uikit/dist/css/components/datepicker.gradient.min.css';
+import 'uikit/dist/css/components/datepicker.almost-flat.min.css';
+import 'uikit/dist/js/components/datepicker.min';
+
+import './styles/style.scss';
```
Comment (dimagreat) [inline app/app.module.js]: It's better to call this folder assets.

File: app/app.module.js [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+import 'uikit/dist/css/uikit.gradient.min.css';
+import 'uikit/dist/css/uikit.almost-flat.min.css';
+import 'uikit/dist/fonts/fontawesome-webfont.ttf';
+import 'uikit/dist/fonts/fontawesome-webfont.woff2';
+import 'uikit/dist/fonts/fontawesome-webfont.woff';
+
+import 'uikit/dist/css/components/datepicker.min.css';
+import 'uikit/dist/css/components/datepicker.gradient.min.css';
+import 'uikit/dist/css/components/datepicker.almost-flat.min.css';
+import 'uikit/dist/js/components/datepicker.min';
```
Comment (dimagreat) [inline app/app.module.js]: It looks weird to have 10 css imports in main.js file, let's move it to assets folder. Maybe we can load big amount of files with array?

File: app/app.module.js [hunk 3]
```diff
@@ -1,12 +1,30 @@
 import angular from 'angular';
 import uiRouter from 'angular-ui-router';
+import ngAnimate from 'angular-animate';
+import ngAria from 'angular-aria';
```
Comment (dimagreat) [inline app/app.module.js]: Do we need these packages with ui.kit ?

File: package.json [hunk 4]
```diff
@@ -23,22 +23,34 @@
   "homepage": "https://github.com/4humans/quiz-app#readme",
   "dependencies": {
     "angular": "1.5.11",
+    "angular-animate": "1.5.11",
+    "angular-aria": "1.5.11",
     "angular-ui-router": "^1.0.3",
     "express": "^4.15.3",
-    "mongodb": "^2.2.28"
+    "jquery": "3.1.1",
```
Comment (dimagreat) [inline package.json]: Do we actually need this?

File: webpack.config.js [hunk 5]
```diff
@@ -23,9 +24,36 @@ const config = {
         ],
       },
       {
+        test: /\.scss$/,
```
Comment (dimagreat) [inline webpack.config.js]: Can we use next syntax?
` loaders:[{
        test: /\.scss$/,
        loaders: ['style', 'css', 'sass']
      },
//...
]
`

File: package.json (hunk 4 above)
Comment (AntonBurduzha) [inline package.json]: y

File: webpack.config.js (hunk 5 above)
Comment (AntonBurduzha) [inline webpack.config.js]: i think we should use config style like in official docs
`{
        test: /\.css$/,
+       use: [
+         {
+           loader: "style-loader",
+         },
+         {
+           loader: "css-loader",
+           options: {
+             modules: true,
+           },
+         },
        ],
      },`

**Pull request closed without merging**